### PR TITLE
Update Generated Test Suites section in README with gem documentation

### LINKED ISSUE #661: README.md: Include documentation on require_all gem
In order to generate a test suite you need the `require_all` gem.
If it's not installed and you try to run `bin/generate <slug>` and get a message like:

```
/Users/deohtee/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- require_all (LoadError)
```
then you probably just need to run `bundle install`.

This should be documented in README.md under [Generated Test Suites](https://github.com/exercism/xruby#generated-test-suites)

### PR DESCRIPTION
**Pull request by deohtee**
## Why?

When I was trying to regenerate the test suites I was getting the error: 
```
/Users/alexanderdelgado/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- require_all (LoadError)
    from /Users/alexanderdelgado/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/alexanderdelgado/Developer/open-source/xruby/lib/generator.rb:1:in `<top (required)>'
    from /Users/alexanderdelgado/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/alexanderdelgado/.rvm/rubies/ruby-2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from bin/generate:4:in `<main>'
```
@Insti let me know that it was because I needed the `require_all` gem and I just needed to run
`bundle install` 

Adding this to the documentation would help others regenerate the test suites successfully in the future. 

## What?

Just added the instructions under the Generating Test Suites section. 
Also fixed a quick typo in the documentation. 

## Screenshots (if appropriate):
screen shot 2017-05-29 at 10 25 37 pm

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or enhancement that would cause existing functionality to change)
- [x] Gardening (code and/or documentation cleanup)

## References and Closures

Closes #661 

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change relies on a pending issue/pull request
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Notes
Let me know if you'd like to see it explained differently or added to another part of the documentation!

### CODE REVIEW

**Comment by Insti** [comment]
I edited the OP to say "Closes #661" so when this is merged the related issue will be automatically closed.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ rake test:clock -- -p
 ```
 
 To run a subset of the tests, use a regular expression. For example, if tests
-exist taht are named identical_to_4_places, and identical, then we can run both
+exist that are named identical_to_4_places, and identical, then we can run both
```
Comment (Insti) [inline README.md]: Good spot 👍

File: README.md [hunk 2]
```diff
@@ -71,6 +71,9 @@ Note that flags which have an attached value, like above, must take the form
 
 Generated test suites use the `bin/generator` cli.
 
+Before using the cli it is recommended you run `bundle install` from within
+the xruby directory to install/update any required gems.
```
Comment (Insti) [inline README.md]: looks good 👍

**Comment by Insti** [comment]
Thanks @deohtee

**Comment by deohtee** [comment]
Thanks again @Insti 😄

**Pull request merged**